testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -0,0 +1,38 @@
+package com.rakuten.tech.mobile.testapp.ui.display
+
+import android.os.Bundle
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.rakuten.tech.mobile.miniapp.MiniAppInfo
+import com.rakuten.tech.mobile.miniapp.testapp.R
+import com.rakuten.tech.mobile.testapp.ui.base.BaseActivity
+import kotlinx.coroutines.launch
+
+class MiniAppDisplayActivity: BaseActivity() {
+
+    private lateinit var viewModel: MiniAppDwnlViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.mini_app_display_activity)
+
+        viewModel = ViewModelProviders.of(this)
+            .get(MiniAppDwnlViewModel::class.java).apply {
+                miniAppView.observe(this@MiniAppDisplayActivity, Observer {
+                    //action: display webview
+                    setContentView(it)
+                })
+            }
+
+        val miniAppInfo = MiniAppInfo(
+            id = "c028be14-4ded-4734-acc6-2d35d9a67630",
```

Comment:
We can do this in another PR, but we need to change `MiniApp#create` to accept `id` and `versionId` instead of `MiniAppInfo` so they don't need to construct a `MiniAppinfo`.